src/utils/AframeRegIndicator.js

```diff
@@ -16,8 +16,8 @@ AFRAME.registerComponent("spotlightindicator",{
         const data = this.data;
 
         /*      define geometry     */
-        let cone = new THREE.CylinderGeometry(.1,.75,1,24,1,true);
```

Comment:
Updated *Geometry to *BufferGeometry since THREE.Geometry was removed in a subsequent release.

Reply:
Another potential area that breaks light indicators